### DISCUSSION TITLE
Material Errors

### DISCUSSION THREAD

**Discussion by nsarifds** [firemodels/fds#13878] [General]
Hello everyone, 

I am trying to run a very complex model in terms of geometry, materials and sensors. I have successfully completed many simulations in the past, so I have a hunch something is going wrong because of the complexity. 

The model has been setup in Pyrosim July Version however I am using CMDfds to run it using FDS 6.9.1.

ERROR(101): Problem with MATL number 4, line number 2032

It is referring to the default PVC material, and specifically, the density line. I have edited the HoC of the material and I am using it for 3 surfaces. Two of the surfaces have the allow obstruction to burn as true, while one of them is also a manually guided reaction. 

&MATL ID='PVC',
      FYI='NISTIR 1013-1 - NIST NRC Validation',
      SPECIFIC_HEAT_RAMP='PVC_SPECIFIC_HEAT_RAMP',
      CONDUCTIVITY_RAMP='PVC_CONDUCTIVITY_RAMP',
      DENSITY=1380.0,
      EMISSIVITY=0.95,
      HEAT_OF_COMBUSTION=3.0E+4/

For troubleshooting purposes, I deleted my 3 surfaces as well as the PVC material. When rerunning the same error appeared. 

ERROR(101): Problem with MATL number 3, line number 2019

Referring to the default CONCRETE material and specifically, the specific_heat line.

&MATL ID='CONCRETE',
      FYI='NBSIR 88-3752 - ATF NIST Multi-Floor Validation',
      SPECIFIC_HEAT=1.04,
      CONDUCTIVITY=1.8,
      DENSITY=2280.0/

All of my CONCRETE surfaces are simple layered surfaces with a thickness and a mass fraction of 1, so nothing weird here. 

Concrete is the first material I have defined, and it is referred to as MATL number 3 while PVC is the second and it is referred to as MATL number 4. I don't know if this is relevant because I only had 4 materials defined in total.

Perhaps it is worth mentioning my mesh size is 0.5m while most of my surfaces are about 0.05m thick. Perhaps something gets messed up when the obstructions snap to the grid?

Is this something that has a known cause? Is there anything that stands out to you as obvious that I am missing? Hopefully it's an easy fix!

**Comment by drjfloyd** [comment]
By July version you mean July of this year? If so then that should be 6.9.1 for Pyrosim.
Are you hand editing the Pyrosim created FDS file or is this error happening with the file as written by Pyrosim?  If it is the file as written by Pyrosim then this imay be an error with Pyrosim.  We would need the simplest version of your input file that still has the error. It will be potentially time consuming for us to have to deal with a multi thousand long input file.  As you simplify it may be that you discover the issue on your own.

**Comment by mcgratta** [comment]
This 101 error simply means that the line is not properly formatted or that one of the variables is not spelled correctly.

**Comment by nsarifds** [comment]
Thank you for confirming the error with the formatting. Combined with the fact this is generated code I was able to track it down. When entering the FYI section in Pyrosim for my custom material it looked like this in the code: FYI=Source, instead of: FYI='Source'. The disorienting thing was the error was pointing to MATL lines that looked normal. Thanks again for your input!
